ql/src/java/org/apache/hadoop/hive/ql/optimizer/calcite/translator/SqlFunctionConverter.java

```diff
@@ -583,6 +584,24 @@ public static SqlAggFunction getCalciteAggFn(String hiveUdfName, boolean isDisti
       CalciteUDFInfo udfInfo = getUDFInfo(hiveUdfName, calciteArgTypes, calciteRetType);
 
       switch (hiveUdfName.toLowerCase()) {
+      case "sketchtoestimate":
+        calciteAggFn =
+            new HiveSqlX(
+                "sketchtoestimate",
+                SqlKind.REGR_SXX,
```

Comment:
it would be better to figure out a way to get rid of these sqlkinds; I guess a standard USER_FUNCTION will just work?